src/Dropdown.js

```diff
@@ -94,6 +94,11 @@ const propTypes = {
    * a menu button.
    */
   role: React.PropTypes.string,
+
+  /**
+   * Which event when fired outside the component will cause it to be closed
+   */
+  rootCloseEvent: React.PropTypes.oneOf(['click', 'mousedown']),
```

Comment:
one of these days i'm going to rename this to `clickOutside` or something... anything really